src/java.base/share/classes/sun/launcher/resources/launcher.properties

```diff
@@ -134,7 +134,7 @@ java.launcher.X.usage=\n\
 \                      append to end of bootstrap class path\n\
 \    -Xcheck:jni       perform additional checks for JNI functions\n\
 \    -Xcomp            forces compilation of methods on first invocation\n\
-\    -Xdebug           does nothing. Provided for backward compatibility.\n\
+\    -Xdebug           deprecated and may be removed in a future release.\n\
```

Comment:
This drops "does nothing" so no longer clear that the option doesn't do anything. It could be changed to "does nothing; deprecated for removal in a future release". Alternatively maybe it would be better to just remove it from the -X output.

Reply:
Hello Alan, I updated the PR to use your suggested wording. I thought it's of no harm to let it be present in the -X output until we actually remove it (soon).